vignettes/tech-dm-join.Rmd

```diff
@@ -157,4 +152,4 @@ dm_nycflights13() %>%
 ```
 
 Be aware that all column names need to be unique.
-The `dm_flatten_to_tbl` cares about automatically renaming the relevant columns and prints if something was changed, e.g. `name -> airlines.name`.
+The `dm_flatten_to_tbl` takes care of this by automatically renaming the relevant columns and informs the user if any names were changed, e.g. `name -> airlines.name`.
```

Comment:
New way of printing renames?

Reply:
what does this comment mean, was it tackled @IndrajeetPatil?

Reply:
Yeah, I am not sure about this means either, so, no, it hasn't been tackled 😅

Reply:
ah ok then let's wait for a clarification by @krlmlr :smiley_cat:

Reply:
@krlmlr I am still awaiting a clarification on this point. Thanks.

Reply:
- The `dm_flatten_to_tbl()` function ...
- The format of the renaming messages `name -> airlines.name` has changed